apps/app/src/views/routes/Settings/components/WebhooksForm.tsx

```diff
@@ -77,6 +77,7 @@ export const WebhooksForm = ({ isOpen, close }: WebhooksFormProps) => {
     'photon:order:fulfillment',
     'photon:order:completed',
     'photon:order:error',
+    'photon:order:canceled',
```

Comment:
There's another spot in the code with the list of webhooks, do we need it there too?